Research/Research/RSDFormUIHint.swift

```diff
@@ -77,11 +77,14 @@ public struct RSDFormUIHint : RawRepresentable, Codable {
     /// Toggle (segmented) button.
     public static let toggle: RSDFormUIHint = "toggle"
     
-    /// Modal step displayed with a secondary button cell.
-    public static let modalButton: RSDFormUIHint = "modalButton"
+    /// Input field of a button-style cell that can be used to display a detail view.
```

Comment:
what's with the weird formatting of 'button' and 'cell' here? is Xcode gettin' all fancy and allowing styled text now, or has Github started formatting code with markdown instead of displaying it as-is, or...?